hack/metrodr-setup.sh

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash
```

Comment:
Other scripts use sh - bash is better but do we really want to maintain both sh and bash scripts?

I would remove both bash and sh and convert use python.